packages/react-components/src/atoms/Pill.tsx

```diff
@@ -89,12 +89,7 @@ const Pill: React.FC<PillProps> = ({
   accent = 'default',
 }) => (
   <span
-    css={({ components }) => [
-      styles,
-      components?.Pill?.styles,
```

Comment:
removing this `components?.Pill?.styles,`
fixes the issue below where the tags are over each other

this is before
![dev gp2 asap science_outputs(Samsung Galaxy S20 Ultra)](https://github.com/yldio/asap-hub/assets/16595804/782f62ba-7e9e-4d4f-83cc-ebe1e8ef8df9)

this is after
![localhost_4000_outputs_4elOfmpAGi24ce2XebHvAp(Samsung Galaxy S20 Ultra)](https://github.com/yldio/asap-hub/assets/16595804/2bae9033-c611-4bd3-94d5-fa35758d58eb)


apparently it didn't affect CRN, I've ran it locally and they still seem good

Reply:
That's great, but those styles were setting the correct font size and other props, but it had` margin: 0` which i believe was the root cause of that issue. so i think it would be nicer to just remove the margin:0 from Pill.styles in `asap-hub/packages/gp2-components/src/templates/components.tsx`